template/template.go

```diff
@@ -205,24 +205,25 @@ func (kv KV) Values() []string {
 // as this will confuse them and prevent simple things like
 // simple equality checks to fail. Map everything to float64/string.
 type Data struct {
```

Comment:
We should add a note here that this is also going to the webhook. Sending things over the network is more expensive than rendering them in memory.